### PR TITLE
[#172124299] Fix obfuscate token and access_token

### PR DESCRIPTION
**Pull request by BurnedMarshal**

### CODE REVIEW

File: src/app.ts [hunk 1]
```diff
@@ -166,7 +166,7 @@ export function newApp(
   );
 
   const obfuscateToken = (originalUrl: string) =>
-    originalUrl.replace(/([?&]token=)[^&]*(&?.*)/, "$1REDACTED$2");
+    originalUrl.replace(/([?&]token=|access_token=)([^&]*)/g, "$1REDACTED");
```
Comment (gunzip) [inline src/app.ts:169]: shouldn't this be:

`([?&]token=|[?&]access_token=)([^&]*)/g`

?

Comment (gunzip) [reply src/app.ts:169]: or we can just use `/(token=|access_token=)([^&]*)/g` ?

Comment (BurnedMarshal) [reply src/app.ts:169]: @gunzip you are right. The correct regex is `([?&]token=|[?&]access_token=)([^&]*)/g`.

Instead `/(token=|access_token=)([^&]*)/g` can't handle correctly the exact parameter name, es. `px_token=123` will be offuscated [example here](https://regex101.com/r/tqBIf1/1)

Comment (BurnedMarshal) [reply src/app.ts]: Fixed in 18b2684

**Pull request merged**